tests/integration/strategy_tests.cpp

```diff
@@ -77,6 +77,47 @@ TEST_P(StrategyTest, Allocate) {
   allocator->deallocate(alloc);
 }
 
+TEST_P(StrategyTest, ReleaseableSizes) {
+  auto strategy = allocator->getAllocationStrategy();
+
+  auto dynamic_pool = std::dynamic_pointer_cast<umpire::strategy::DynamicPool>(strategy);
+
+  if (!dynamic_pool) {
+    std::cerr << allocator->getName() << " is not a DynamicPool, cannot run this test!\n";
+    return;
```

Comment:
@davidbeckingsale , I have thus far not been able to get a non-null value for dynamic_pool.  Can you take a look and help me understand what I might be doing wrong here?